pipelines/SpeciesTreeFromBusco/nextflow.config

```diff
@@ -18,10 +18,11 @@ includeConfig "$ENSEMBL_ROOT_DIR/ensembl-compara/pipelines/nextflow.config"
 
 params {
     dir = "$HPS_HOME/Workspace/busco_tree/species_fasta"
+    gzipped = "true"
```

Comment:
Are you still using this? I cannot find it anywhere in this commit?

Reply:
No, this slipped through. Not using it actually so will remove it.